pkgs/applications/office/libreoffice/plugins.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, ruby.devEnv, zip }:
```

You'll probably need something like
```suggestion
{ stdenv, fetchFromGitHub, ruby, zip }:
```